captum/attr/_core/neuron/neuron_deep_lift.py

```diff
@@ -202,13 +209,15 @@ def __init__(self, model, layer):
 
     def attribute(
         self,
-        inputs,
-        neuron_index,
-        baselines,
-        additional_forward_args=None,
-        attribute_to_neuron_input=False,
-        custom_attribution_func=None,
-    ):
+        inputs: TensorOrTupleOfTensors,
+        neuron_index: Union[int, Tuple[int, ...]],
+        baselines: Optional[
+            Union[int, float, Tensor, Tuple[Union[Tensor, int, float], ...]]
```

Comment:
This baselines (only for NeuronDeepLiftShap) Union should also include Callable[..., Union[Tensor, Tuple[Tensor,...]]]

Reply:
Thanks!